Bugfix for bug 241711 (TimberServiceClass)

Rebecca-Coding21 opened the pull request:
# Description

I fixed a bug that occured while trying to create a TimberServiceClass(). There was a typo (member instead of members) in the code of the HLF.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Unit Tests
- [x] Attached examples
[TimberServiceClass_Test.zip](https://github.com/user-attachments/files/16452560/TimberServiceClass_Test.zip)

**Test Configuration**:
* RFEM version: 6.07.0005
* Python version: 3.10.11


# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

heetrojivadiya commented on RFEM/TypesForTimberDesign/timberServiceClass.py:
```diff
@@ -39,7 +39,7 @@ def __init__(self,
 
         # Assigned Members
-        clientObject.member = ConvertToDlString(members)
+        clientObject.members = ConvertToDlString(members)
```
![image](https://github.com/user-attachments/assets/f4c37951-70af-488a-a39a-503a1265486d)

The test is getting failed with above log.

heetrojivadiya reviewed (commented):
It would be great if you add the same changes in [timberMoistureClass.py](https://github.com/dlubal-software/RFEM_Python_Client/blob/main/RFEM/TypesForTimberDesign/timberMoistureClass.py) and [timberServiceCondition.py](https://github.com/dlubal-software/RFEM_Python_Client/blob/main/RFEM/TypesForTimberDesign/timberServiceCondition.py).

Rebecca-Coding21 commented:
I added the same changes to timberMoistureClass and timberServiceCondition and updated test_timberMoistureClass as well. The tests for timberServiceCondition don't need an update.
